extensions/wikia/GlobalWatchlist/GlobalWatchlistHooks.class.php

```diff
@@ -8,16 +8,10 @@ public static function getPreferences( $user, &$defaultPreferences ) {
 
 		$defaultPreferences['watchlistdigest'] = [
 			'type' => 'toggle',
-			'label-message' => 'tog-watchlistdigest',
-			'section' => 'watchlist/advancedwatchlist',
+			'label-message' => 'tog-watchlistdigest-v2',
+			'section' => 'emailv2/email-me-v2',
```

Comment:
By updating these `section` and `labe-message` values here, we can remove the code which is overriding these values [here](https://github.com/Wikia/app/pull/8944/files#diff-19b0d1c2e76c0dcf56b1bdf2b0eeab7cL160).